src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -289,11 +314,15 @@ int main(int argc, char *argv[]) {
 
 
     int32 srand_seed = 0;
+    std::string fill_mode_string = "nearest";
 
     ImageAugmentationConfig config;
 
     ParseOptions po(usage);
     po.Register("srand", &srand_seed, "Seed for the random number generator");
+    po.Register("fill-mode", &fill_mode_string, "Mode for dealing with "
```

Comment:
Shouldn't this option be defined in the ImageAugmentationConfig class? Is it here because it is a string and is converted to Enum later?